csuntechlab/iSTART
# STR-38 -- FE: Create middleware for routes.js to return user to a 404 error page

# Description
Created a redirect for non-existing routes to the Error.vue

# Testing
1. Visit the /about route to ensure the route is working.
2. Visit a non-existing route and make sure a 404 page appears.
